examples/README.md

```diff
@@ -26,8 +26,8 @@ $ npm install
 ## Table of contents
 
 - #### Authentication
-  - [Sense using ticketing](./authentication/sense-using-ticketing)
```

Comment:
Should we still provide an example for ticketing? Otherwise we should close that issue.